tests/benchmark/plot.py

```diff
@@ -0,0 +1,29 @@
+import matplotlib.pyplot as plt
+import subprocess
+
+ret = [l.split() for l in subprocess.check_output(['./time.sh']).decode('utf-8').splitlines()]
```

Comment:
How about we completely remove the shell script and instead do all the execution and measurements directly within Python?

Reply:
i've done as you've suggested. could you take another look?